src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/Services/SemrushCachingService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+using Newtonsoft.Json;
+
+using Umbraco.Cms.Integrations.SEO.SemrushTools.Models.Dtos;
+using Umbraco.Core.Cache;
+
+namespace Umbraco.Cms.Integrations.SEO.SemrushTools.Services
+{
+    public class SemrushCachingService: ISemrushCachingService<RelatedPhrasesDto>
```

Comment:
If you follow the suggestion above this would become `CacheHelper : ICacheHelper`.

Reply:
Item addressed.